### DISCUSSION TITLE
Nanogate Overhaul and Rebalance

### DISCUSSION THREAD

**Discussion by EmilitiaEnnehrt** [Equinox-SS13/equinox-sojourn#23] [Ideas]
It is my personal opinion is that the Nanogate is the simpleman's core implant that bypass the whole level up grind process.
* You're sacrificing stat level up to having your base do things a bit better/more unique than just having stats.

Currently, the problem is that compared to the cruciform and psionic organ is that it is woefully underpowered, both Artificer's, Opifex (although substituted with Opifex having their own unique perks), and normal nanogate.
>An argument may be made about how Nanogate is not as powerful when you are not playing the intended job (Artificer), but the same cannot be said for Cruciform and Psionic. The Psionic organ retain all of it's content but at a weaker resource pool compared to Soteria, and the Cruciform still maintain three pages of rite wherever you are, and still get buffed periodically by the Will of the Protector.

>Another note is that most options within the Nanogate are straight up inferior version of other flavored options that the two core implants have; cruciform and psionic can remotely send messages to other people, the nanogate has an email pop up, cruciform and psionic can stabilize/heal themselves while nanogate only does at 0.2 brute/burn per second, nanite RIG (which is a point sink) is worse Psionic armor, etc.

A good look at it too is that the relatively high cost means that you can only choose 2-4 options at a time, making different nanogate users distinct from each other about what their lore-wise "core implant", allowing for more roleplay option.
* some players played it off as spine replacement, some people play it off like part of their brain @zenitheevee (should check this), some people play it as them having custom implant or species features..

The problem I have with it (and other people) is that the choices suck right now, not because they're all useless but because they're all nerfed to hell after the medical update (everyone started using nanogate for the 1 brute/burn heal because nobody could use med)
* What I think would help as an overhaul is rebalancing how existing module works /and/ adding new, subtle but unique and inferior options that some of the cruciform have (since they are both lore-wise and functionality-wise similar, but get power from different source)
* What I think would NOT help as an overhaul is making having a Nanogate require a form of "working for", the Cruciform doesn't (arbet regenerate faith and has cooldown + require bible to use), but having Nanogate require resources or a ton of engagement to make it remotely good would go completely against the philosophy of taking it to get flat bonus over having to grind for insight level up.

A few more silly ideas.
* Swap point cost for the speed buff (muscles) and brute reduction (skin weave) since speed isn't that useful here in PvE and already cost 5 points
* Have an option for things to be able to thrown farther, faster, at small cost.
* Have an option for further leaps
* Have an option that heals only toxin at medium cost, would not be compatible alongside brute/burn regen
* Have an option that mildly heal pain, making paincrit ceiling a little higher.

**Comment by Kilmented** [comment]
all I use my nanogates for are 2 things
1. nanosuit RIG, specifically for 4 things
A. the boots (they have completely silent running and integrated magboots)
B. the helmet has a really good flashlight
C. rigsuit laser (double DPS)
D. autodoc
2. having a max upgraded tablet in my head at all times
'nuff said it's just useful
all the other abilities are useful in some scenarios but complete dogshit in others
choosing 
rig -> autodoc -> laser 
puts you in a generalist position, lets you ignore medical outright for minor organ damage, makes you immune to flashes, makes you immune to 90% radiation, makes you immune to biotic things


at the following costs

reduced inspirations (stats come with your job anyways so it doesns't fucking matter)
NO BACKPACK. GET GOOD AT INVENTORY MANAGEMENT.
severe EMP vulnerability



the extra points from the guildie one just doesn't matter to me (I played prospector) and the extra stuff from the opifex flies over my head because I am not playing a bird (evil government drone) in any circumstance

tl;dr
nanogate has only one real downside if you choose rig -> autodoc -> laser because vig as a stat doesn't feel like it matters for combat encounters (I would play negative Vig if I could), that being not starting with a backpack

**Reply by Kilmented** [reply]
im sure if you had better armor or worse stats to start out with, there'd be different picks, but Salvagers start with expert in cog, mech, bio midgame armor and midgame weapons, so there's literally no reason why I should ever get MORE passive damage mitigation and instead focus on more ACTIVE damage mitigation

it works well enough I think
(five minutes in greyson)
![image](https://github.com/user-attachments/assets/dbabfe6c-57e9-4da1-827a-01faad18b3ba)
((average greyson trip))
https://cdn.discordapp.com/attachments/1125646663797719040/1271273854567841875/N27NazbfKWC9Is3l.mp4?ex=66b8b7ca&is=66b7664a&hm=a1e781d2eb7b5e7eb1ca91c2d7645095c0d0d5fd3aee05380ceff2cf4d7cbdd8&

**Comment by Andron1219** [comment]
Suggestion to make it feel less of a damaging item as well; Reduce the penalty for inspiration gain from it. If it's not that good, it shouldnt completely axe an entire feature of Eriscode to begin with.
